### ISSUE TITLE
play 2.4.x

### ISSUE THREAD

**Issue by ergomesh** [pvoznenko/play-slick-angular-test-example#1]
I would be interested to see how this would be upgraded to work for play2.4.x and slick3

**Comment by pvoznenko** [comment]
Hi @ergomesh, good point. 

Will update it when I have some free time.

Cheers!

**Comment by ergomesh** [comment]
Cool I tried but couldn't get the getFilteredQuery and filterRowBased working correctly in slick3. Also, I wasn't sure the best method in slick3 for making it object of the ReportsModel considering the injection and futures in 2.4.x 

Fairly different beast.

**Comment by pvoznenko** [comment]
Yep, they totally changed way on how to work with slick 3 compare to prev 2nd version. So it will be not 5 min work. Also I might consider some architecture changes too :)
